src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,289 @@
+    display: flex;
+    font-family: 'Montserrat-700';
+    font-size: 14px;
+    line-height: 17px;
+    align-items: center;
+    letter-spacing: 0.04em;
+    color: #000000;
+`;
+
+export const MenuSection = styled.div`
+    ${elementMenu}
+    border-right: 1px solid #858598;
+`;
+
+export const MenuInitiaLetterCircle = styled.div`
+    background: #2196F3;
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    margin-left: 24px;
+    display: grid;
+    align-items: end;
+    @media (max-width: 531px){
+        margin-left:20px;
+    }
+`;
+
+export const MenuInitiaLetter = styled.div`
+    width: 26px;
+    height: 24px;
+    font-family: 'Montserrat-700';
+    font-size: 17px;
+    line-height: 22px;
+    display: -ms-flexbox;
+    letter-spacing: 0.04em;
+    color: #FFFFFF;
+    display: grid;
+    align-items: end;
+    justify-content: center;
+`;
+
+export const MenuName = styled.div`
+    margin-left: 8px;
+    padding-right: 16px;
+    height: 19px;
+    display: grid;
+    align-items: center;
+    font-family: 'Montserrat-700';
+    font-size: 12px;
+    line-height: 15px;
+    letter-spacing: 0.04em;
+    border-right: 1px solid #858598;
+    color: #858598;
+    @media (max-width: 531px){
+        border-right: 1px solid white;
+        color: white;
+    }
+`;
+
+// ----------- styled for - HeaderMobile ----------
+export const MobileHamburgerLine = styled.div`
+    border-bottom: 1px solid #858598;
+`;
+
+export const MobileHamburgerContainer = styled.nav`
+    position: relative;
+    width: 22px;
+    height: 14px;
+    border-top: 1px solid #858598;
+    display: flex;
+    flex-direction: column;
+    border-bottom: 1px solid #858598;
+    justify-content: center;
+    cursor: pointer;
+    &:hover {
+        width: 23px;
+        height: 15px;
+        border-top: 1px solid black;
+        border-bottom: 1px solid black;
+        box-shadow:         
+        rgb(0 0 0 / 19%) 1px 3px 20px 0px, 
+        inset -10px -10px 15px rgb(255 255 255 / 80%), 
+        inset 10px 10px 15px rgb(70 70 70 / 12%);
+        ${MobileHamburgerLine}{
+            border-bottom: 1px solid black;
+        }
+    }
+    @media (max-width: 300px){
+        margin-left:20px;
+    }
+`;
+
+// -------------------------------------------------
+export const MobileHamburgerbackground = styled.nav`
+    position: fixed;
+    width: 274px;
+    height: 100%;
+    right: 0px;
+    top: 0px;
+    background: #FFBC33;
+    box-shadow: -15px 0px 20px rgba(0, 0, 0, 0.25);
+    Z-index: 10;
+    overflow: hidden;
+    transition: 0.5s;
+`;
+
+export const MobileHamburgerSectionStyle = css`
+    height: 44px;
+    width: 100%;
+    border-bottom: 1px solid #ffffff;
+    display: grid;
+    align-items: center;
+    padding-left: 20px;
+    ${fontRegular}
+    &:hover {
+    box-shadow: 
+    rgb(0 0 0 / 19%) 1px 3px 10px 0px, 
+    inset -6px -6px 8px rgb(255 255 255 / 80%), 
+    inset 6px 6px 8px rgb(70 70 70 / 12%);
+  }
+`;
```

Comment:
please keep hover styles according to mockup

Reply:
I will remove this hover.